app/src/main/java/org/stepik/android/view/lesson/ui/activity/LessonActivity.kt

```diff
@@ -290,6 +289,8 @@ class LessonActivity : FragmentActivityBase(), LessonView,
         isInfoMenuItemVisible =
             state is LessonView.State.LessonLoaded &&
             state.stepsState is LessonView.StepsState.Loaded
+
+        centeredToolbarSubtitle.isVisible = stepsAdapter.items.isNotEmpty()
```

Comment:
maybe we should have this line only in `is LessonView.State.LessonLoaded` branch